aptos-move/framework/aptos-framework/sources/Voting.move

```diff
@@ -141,6 +148,7 @@ module AptosFramework::Voting {
     /// @param expiration_secs The time in seconds at which the proposal expires and can potentially be resolved.
     /// @return The proposal id.
     public fun create_proposal<ProposalType: store>(
+        proposer: address,
```

Comment:
do we want to be signer? or we expect proposer and signer can be different?

Reply:
This doesn't need to be a signer because only the governance module can call create_proposal (ProposalType is required as a param).